carbonserver/render.go

```diff
@@ -221,7 +221,7 @@ func (listener *CarbonserverListener) renderHandler(wr http.ResponseWriter, req
 		return
 	}
 
-	if response.metricsFetched == 0 {
+	if response.metricsFetched == 0 && listener.emptyResultOk == false  {
```

Comment:
Second Deepsoure warning has sense. `&& listener.emptyResultOk == false` should be better replaced with `&& !listener.emptyResultOk`